Assets/Scripts/Units/Melee Unit/MeleeUnit.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RogueApeStudio.Crusader.Units.MeleeUnit
+{
+    public class MeleeUnit : MonoBehaviour
+    {
+        public UnitMovement LocalUnitMovement;
```

Comment:
Does not conform to our code standard for fields.
- Public
- Lacks underscore prefix.
- PascalCase

Reply:
Unresolved.